docs/setup.md

```diff
@@ -54,6 +54,7 @@ Copy the `config.sample.yml` to a new file `config.yml`. The sample config is al
 [here](./setup/sample-configuration.md) for your convenience.
 
 You should read and fill this in as the bridge will not start without a complete config.
+Sections and items marked as `Optional` need to either be configured if you want to use the respective feature, or removed/commented out to disable the feature.
```

Comment:
I want to be careful not to suggest the commenting out disables things, because that's not universally true.

```suggestion
Sections and items marked as `Optional` can either be configured if you want to use the respective feature, or removed/commented out. When items are commented out, the defaults are used.
```